From 11b595e5053115a726c075dcc5c9eddf8caaff7e Mon Sep 17 00:00:00 2001
From: Piotr Dymacz <pepe2k@gmail.com>
Date: Tue, 30 May 2017 22:25:02 +0200
Subject: [PATCH] ar71xx: image: drop redundant uboot-envtools from
 DEVICE_PACKAGES

uboot-envtools is already included in DEFAULT_PACKAGES for ar71xx.

Signed-off-by: Piotr Dymacz <pepe2k@gmail.com>
---
 target/linux/ar71xx/image/generic.mk | 12 ++++--------
 1 file changed, 4 insertions(+), 8 deletions(-)

diff --git a/target/linux/ar71xx/image/generic.mk b/target/linux/ar71xx/image/generic.mk
index d978dec412..29ed108d5d 100644
--- a/target/linux/ar71xx/image/generic.mk
+++ b/target/linux/ar71xx/image/generic.mk
@@ -140,7 +140,7 @@ TARGET_DEVICES += gl-ar300
 
 define Device/gl-ar300m
   DEVICE_TITLE := GL AR300M
-  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 uboot-envtools
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2
   BOARDNAME = GL-AR300M
   IMAGE_SIZE = 16000k
   CONSOLE = ttyS0,115200
@@ -259,7 +259,6 @@ TARGET_DEVICES += wndr3700 wndr3700v2 wndr3800 wndr3800ch wndrmac wndrmacv2
 
 define Device/cap324
   DEVICE_TITLE := PowerCloud CAP324 Cloud AP
-  DEVICE_PACKAGES := uboot-envtools
   BOARDNAME := CAP324
   DEVICE_PROFILE := CAP324
   IMAGE_SIZE = 15296k
@@ -270,7 +269,6 @@ TARGET_DEVICES += cap324
 
 define Device/cap324-nocloud
   DEVICE_TITLE := PowerCloud CAP324 Cloud AP (No-Cloud)
-  DEVICE_PACKAGES := uboot-envtools
   BOARDNAME := CAP324
   DEVICE_PROFILE := CAP324
   IMAGE_SIZE = 16000k
@@ -281,7 +279,6 @@ TARGET_DEVICES += cap324-nocloud
 
 define Device/cr3000
   DEVICE_TITLE := PowerCloud CR3000 Cloud Router
-  DEVICE_PACKAGES := uboot-envtools
   BOARDNAME := CR3000
   DEVICE_PROFILE := CR3000
   IMAGE_SIZE = 7104k
@@ -292,7 +289,6 @@ TARGET_DEVICES += cr3000
 
 define Device/cr3000-nocloud
   DEVICE_TITLE := PowerCloud CR3000 (No-Cloud)
-  DEVICE_PACKAGES := uboot-envtools
   BOARDNAME := CR3000
   DEVICE_PROFILE := CR3000
   IMAGE_SIZE = 7808k
@@ -303,7 +299,7 @@ TARGET_DEVICES += cr3000-nocloud
 
 define Device/cr5000
   DEVICE_TITLE := PowerCloud CR5000 Cloud Router
-  DEVICE_PACKAGES := uboot-envtools kmod-usb2 kmod-usb-ohci kmod-usb-ledtrig-usbport kmod-usb-core
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-ohci kmod-usb-ledtrig-usbport kmod-usb-core
   BOARDNAME := CR5000
   DEVICE_PROFILE := CR5000
   IMAGE_SIZE = 7104k
@@ -314,7 +310,7 @@ TARGET_DEVICES += cr5000
 
 define Device/cr5000-nocloud
   DEVICE_TITLE := PowerCloud CR5000 (No-Cloud)
-  DEVICE_PACKAGES := uboot-envtools kmod-usb2 kmod-usb-ohci kmod-usb-ledtrig-usbport kmod-usb-core
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-ohci kmod-usb-ledtrig-usbport kmod-usb-core
   BOARDNAME := CR5000
   DEVICE_PROFILE := CR5000
   IMAGE_SIZE = 7808k
@@ -690,7 +686,7 @@ endef
 
 define Device/dap-2695-a1
   DEVICE_TITLE := D-Link DAP-2695 rev. A1
-  DEVICE_PACKAGES := ath10k-firmware-qca988x kmod-ath10k uboot-envtools
+  DEVICE_PACKAGES := ath10k-firmware-qca988x kmod-ath10k
   BOARDNAME = DAP-2695-A1
   IMAGES := factory.img sysupgrade.bin
   IMAGE_SIZE = 15360k
-- 
2.13.1

